templates/hello.html

```diff
@@ -8,7 +8,7 @@
 </head>
 <body>
 <h2><a href="https://github.com/SwagLyrics/swaglyrics-backend">SwagLyrics Backend</a></h2>
-<p>The SwagLyrics Backend and API is housed here. This content should be more descriptive.</p>
+<p>The SwagLyrics Backend and API is housed here. A key function of the Backend is adding unsupported songs to the list below as well as opening <a href ="https://github.com/SwagLyrics/SwagLyrics-For-Spotify/labels/unsupported%20song">GitHub Issues</a> for these songs. The backend also checks these songs with the Genius API and confirms the validity of data it is provided. Click the above link to explore the backend on GitHub!</p>
```

Comment:
Nice! Can you make it span multiple lines instead of one line for readability? (I don't mean insert <br> tags just the enter key so source code is more readable)

Reply:
Sure!